pkgs/test_api/lib/src/backend/suite_platform.dart

```diff
@@ -52,7 +52,9 @@ class SuitePlatform {
   factory SuitePlatform.deserialize(Object serialized) {
     var map = serialized as Map;
     return SuitePlatform(Runtime.deserialize(map['runtime'] as Object),
-        compiler: Compiler.deserialize(map['compiler'] as Object),
+        compiler: map.containsKey('compiler')
```

Comment:
why not just `as Object?`

Reply:
I would have to change the definition of `Compiler.deserialize` for that.